components/content-search/index.js

```diff
@@ -357,6 +359,8 @@ const ContentSearch = ({
 					onBlur={() => {
 						setIsFocused(false);
 					}}
+					size="compact"
+					style={{ border: '1px solid rgb(30, 30, 30)' }}
```

Comment:
I'm not sure we need this manual border. With the greater space it has this should look better without I think

Reply:
My concern here is that I cannot see the border of the input at all - maybe it's my eyes, but I don't even realize it's a search input without the border

Reply:
Ah - there was something with my monitor - probably a high contrast setting (ironic enough) that was preventing me from seeing the gray background of the input. I've removed the manual style.